cmd/example/main.go

```diff
@@ -59,7 +59,7 @@ func main() {
 
 	electionID := p4_v1.Uint128{High: 0, Low: 1}
 
-	p4RtC := client.NewClient(c, deviceID, electionID, nil)
```

Comment:
you need to undo all the changes under `cmd/` now that we have added `NewClientForRole` (no change to `NewClient`)

Reply:
Are you looking at the latest changes? This seems from from an older commit.

Reply:
yes, that part looks good now